.github/workflows/test-mobile-e2e-reusable.yml

```diff
@@ -121,10 +126,21 @@ jobs:
       - name: Build JS Bundle app for Detox test run
         if: steps.detox-build.outputs.cache-hit == 'true'
         run: pnpm mobile e2e:ci -p ios --bundle
+      - name: Setup Speculos image and Coin Apps
```

Comment:
Why do we want to include Speculos on all e2e Detox tests? Don't we want to have a run with Speculos and one without? I'm afraid that this will increase the feedback loop when we open a Pull Request, wdyt?

Reply:
If you check the next line, it will only be run for speculos_tests

Reply:
Here it is : [Link](https://github.com/LedgerHQ/ledger-live/actions/runs/11953194465)